### PR TITLE
Feature/issue 29

### PR DESCRIPTION
**Pull request by HeyItsMario**
Created the Table component and its documentation. I also upgraded marked 0.3.6 to 0.3.9

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+                "program": "enter program name, for example ${workspaceRoot}/a.out",
+                "args": [],
+                "stopAtEntry": false,
+                "cwd": "${workspaceRoot}",
+                "environment": [],
+                "externalConsole": true,
+                "MIMode": "lldb"
+            }
+        ]
+    }
```
Comment (kevmojay) [inline .vscode/launch.json]: vscode config files, shouldn't be in the repo

**Comment by HeyItsMario** [comment]
I decided to just wrap all the individual HTML's table components into our own MetaTable component. That way it looks more clear and resembles regular html structure.

File: src/components/meta/Table.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { generate as shortIdGenerate } from 'shortid';
```
Comment (kevmojay) [inline src/components/meta/Table.jsx]: shortIdGenerate unused

File: src/components/meta/Table.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+};
+
+
+MetaTable.defaultProps = {
+  className: '',
+  header: [],
+};
+
+MetaTable.propTypes = {
+  className: PropTypes.string,
```
Comment (kevmojay) [inline src/components/meta/Table.jsx]: className prop unused. Should it be used?

**Pull request merged**